content/en/platform/corda/4.8/open-source/key-concepts-time-windows.md

```diff
@@ -31,45 +31,40 @@ title: Time-windows
 
 ## Time in a distributed system
 
-A notary also act as the *timestamping authority*, verifying that a transaction occurred during a specific time-window
-before notarising it.
+The [notary cluster](../../../../../en/platform/corda/4.8/open-source/key-concepts-notaries.md) acts as the *timestamping authority*.
+It verifies that a transaction occurred during a specific time-window before notarizing it.
 
-For a time-window to be meaningful, its implications must be binding on the party requesting it. A party can obtain a
-time-window signature in order to prove that some event happened *before*, *on*, or *after* a particular point in time.
-However, if the party is not also compelled to commit to the associated transaction, it has a choice of whether or not
-to reveal this fact until some point in the future. As a result, we need to ensure that the notary either has to also
-sign the transaction within some time tolerance, or perform timestamping *and* notarisation at the same time. The
-latter is the chosen behaviour for this model.
+A party obtains a time-window signature in order to prove a transaction happened *before*, *on*, or *after* a particular point in time.
+As long as the party is not required to commit to the associated transaction, it can choose to reveal this fact at some point in the future. As a result the notary
+timestamps and notarizes at the same time.
 
-There will never be exact clock synchronisation between the party creating the transaction and the notary.
-This is not only due to issues of physics and network latency, but also because between inserting the command and
-getting the notary to sign there may be many other steps (e.g. sending the transaction to other parties involved in the
-trade, requesting human sign-off…). Thus the time at which the transaction is sent for notarisation may be quite
-different to the time at which the transaction was created.
+Thus, the time at which the transaction is sent for notarization may be
+different to the transaction creation time. There will never be exact clock synchronization between the party creating the transaction and the notary for two reasons:
```

Comment:
This reordering confused me a bit - it made it seem like the fact that the node can reveal the fact later affects the time perception between the node and notary, but I think it might be something more like this:
```suggestion
A node may not send a transaction to the notary right away—they might need to circulate the transaction to other nodes involved in the transaction, or request human sign-off. Even if the node sends it as soon as it's generated, the node's clock and the notary's clock will never be perfectly in sync due to latency and physics. This means that the timestamp on a transaction is usually different from the time it was created.  
```

Reply:
I also think this paragraph belongs a bit further down — at this point it's not clear why we would care about a tiny lag, but later you learn that this is the reason for time-windows rather than exact times.